api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1184,6 +1184,14 @@ message Cluster {
   // If ``connection_pool_per_downstream_connection`` is true, the cluster will use a separate
   // connection pool for every downstream connection
   bool connection_pool_per_downstream_connection = 51;
+
+  // The idle timeout for connections. The idle timeout is defined as the period in which
+  // there are no active stream. When the idle timeout is reached the connection will be closed.
+  //
+  // .. warning::
+  //   Disabling this timeout has a highly likelihood of yielding connection leaks due to lost TCP
```

Comment:
Is this then on by default?  If so we should comment on the default timeout and runtime guard the behavioral change.

Reply:
Yes I'll add a runtime guard. Thanks for pointing this out.